src/finalize

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+#
+# Final steps to run before finishing installation.
+#
+
+# Install the latest long-term support version of npm. In particular, this is
+# needed for builds on tree-sitter grammars that use newer Javascript features.
+npm install -g n
```

Comment:
Why we can't do that instead in our Dockerfile and Makefile in semgrep and ocaml-tree-sitter-semgrep?

Reply:
i preferred upgrading our version of `nodejs` in all our images than updating on a per-project basis, because it catches the problem further upstream